webapp/html/columns.html

```diff
@@ -36,11 +36,11 @@ <h3 class="display-4 text-center">{{ file }}</h3>
     <div class="col reportSentencesDiv">
         {% for elmt in final_html %}
             {% if elmt.tag == 'img' %}
-                <img src="{{elmt.text}}" class="normalImage">
+                <img src="{{elmt.text}}" class="normalImage" onclick="sentenceContext({{elmt.uid}}, true)">
```

Comment:
I think we should being sending a type rather than true or false here. Down the line we might have other html tags we want to identify and send to the function